src/software/power/pins.h

```diff
@@ -24,3 +24,13 @@ const uint8_t BREAK_BEAM_PIN = 37;
 // UART
 const uint8_t RXD2 = 19;
 const uint8_t TXD2 = 22;
+
+// Power Monitor
+const uint8_t PM_SDA = 13;
+const uint8_t PM_SCL = 14;
+
+// Timers
+const uint8_t CHICKER_TIMER = 0;
+const uint8_t GENEVA_TIMER = 1;
+const uint8_t CHARGE_TIMER = 2;
+const uint32_t MICROSECONDS_IN_SECOND = 1000000;
```

Comment:
can we use the value already in src/shared/constants.h?

Reply:
not currently if we update constants to be constexpr then we could